mod/photos.php

```diff
@@ -1139,7 +1139,11 @@ function photos_content(App $a)
 						'$preview' => DI::l10n()->t('Preview'),
 						'$loading' => DI::l10n()->t('Loading...'),
 						'$qcomment' => $qcomment,
-						'$rand_num' => Crypto::randomDigits(12)
+						'$rand_num' => Crypto::randomDigits(12),
+						// Dropzone
+						//'$max_imagesize'       => DI::config()->get('system', 'maximagesize'),
+						// don't know, if DI::config.. does not work here, so it is set to a manual value
```

Comment:
`DI::config()` should work here.

Reply:
it should... but when use it, i get http 500...

Reply:
These should produce an error message in your Friendica log that you should enable when you work on it. Notice level should be enough.

Reply:
I also changed it to DI, becaus ist listed in the use-headers.